athenahealth/httpclient.go

```diff
@@ -164,7 +97,7 @@ func (h *HTTPClient) setBaseURL() {
 func (h *HTTPClient) request(ctx context.Context, method, path string, body io.Reader, headers http.Header, out interface{}) (*http.Response, error) {
 	if _, ok := ctx.Deadline(); !ok {
```

Comment:
Not blocking but how would we feel about an explicit test case for this (`request` called with a context with Deadline already set to override default) since it tripped us up on this PR and we already have at least one test for `request`?

Reply:
good call - added tests https://github.com/eleanorhealth/go-athenahealth/pull/109/commits/fde526f74be55a88a8751af5c857c6a327258619